cimsparql/model.py

```diff
@@ -234,7 +234,7 @@ def loads(
         station_group_optional: bool = True,
         station_group: bool = False,
         with_sequence_number: bool = False,
-        network_analysis: bool = True,
+        network_analysis: Optional[bool] = True,
```

Comment:
**Spm**: Hva er det man oppnår med `Optional[bool]` i forhold til `bool`? Jeg mener, `bool` har to verdier `True/False`. Er det bare at man kan ha `None` i tillegg? I så fall hva er forskjellen på `False` og `None` i disse funksjonene?